backend/src/apiserver/server/visualization_server.go

```diff
@@ -115,13 +112,14 @@ func (s *VisualizationServer) getVisualizationServiceURL(request *go_client.Crea
 	return s.serviceURL
 }
 
-func isVisualizationServiceAlive(serviceURL string) bool {
+func isVisualizationServiceAlive(serviceURL string) (bool, error) {
```

Comment:
nit: returning the bool is now kind of redundant. We can just return the error. If error is not nil, it failed.